src/adm/settings.rs

```diff
@@ -307,7 +308,11 @@ impl AdmFilterSettings {
             })?
             .to_string();
         let path = settings_bucket.path().trim_start_matches('/');
-        let contents = cloud_storage::Object::download(&bucket_name, path)
+        let req = reqwest::Client::builder()
```

Comment:
forgot connect_timeout here as well, looks like you added it to all the tests where it doesn't matter much.

Reply:
Yeah, just wanted to be consistent.